.github/workflows/ci-playwright.yml

```diff
@@ -0,0 +1,57 @@
+name: CI - Playwright
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
```

Comment:
Quotes are not needed and this then will be consistent with other branches used on GitHub Actions

Reply:
oaky. should we run this on schedule too right? weekly maybe?